mmdeploy/codebase/mmdet3d/deploy/voxel_detection.py

```diff
@@ -268,7 +325,7 @@ def single_gpu_test(self,
         results = []
         dataset = data_loader.dataset
 
-        prog_bar = mmcv.ProgressBar(len(dataset))
+        prog_bar = mmengine.ProgressBar(len(dataset))
```

Comment:
`single_gpu_test`  method should be removed.

Reply:
done.